companyweb_base/models/res_partner.py

```diff
@@ -547,15 +551,16 @@ def cweb_button_copy_address(self):
             message=_("Companyweb Address Successfully copied")
         )
 
-    def _cweb_call_wizard_credentials(self, wizard_name):
+    @api.model
+    def _cweb_call_wizard_credentials(self, wizard_name, context):
```

Comment:
accepting the context as argument sounds weird ? Why not use `self.env.context` ?